playground/frontend/lib/locator.dart

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:app_state/app_state.dart';
+import 'package:get_it/get_it.dart';
+import 'package:playground_components/playground_components.dart';
+
+import 'config.g.dart';
+import 'pages/loading/page.dart';
+import 'router/page_factory.dart';
+import 'router/route_information_parser.dart';
+
+Future<void> initializeServiceLocator() async {
```

Comment:
I think `main.dart` should be as short as possible because it conveys no meaning. In this file, the file name is another hint of what it does. And also this way files have less reasons to change.

As for static methods, see above.